test/unit/ngsiv2/contextRequests/createBidirectionalDevice.json

```diff
@@ -2,7 +2,7 @@
   "id": "Second MQTT Device",
   "type": "AnMQTTDevice",
   "location": {
-    "type": "geo:point",
-    "value": "0, 0"    
+    "type": "geo:json",
+    "value": {"type":"Point","coordinates":[0,0]}
```

Comment:
Shouldn't be unit test failing before this fix (I mean, in master)?

Reply:
If you re-start the last run it should now fail. The change has occurred in the downstream iotagent-node-lib

"iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
... is now picking up telefonicaid/iotagent-node-lib@a7eb260 commited 20 days ago, but the last CI test run was 23 days ago.

Reply:
Re-run in master and failed. Everything ok.

NTC